hw/ip/aes/pre_sca/alma/README.md

```diff
@@ -0,0 +1,151 @@
+# AES Formal Masking Verification Using Alma
+
+This directory contains support files to formally verify the masking employed
+inside AES unit using the tool [Alma:
+Execution-aware Masking Verification](https://github.com/IAIK/coco-alma).
+
+## Prerequisites
+
+1. Download the Alma tool
+
+   Note that since we are primarily interested in verifying the masking of a
+   hardware implementation, we are using the `hw-verif` branch of the tool. In
+   addition, we currently need to use a patched version of the tool, to work
+   around a limitation.
+   ```sh
+   git clone git@github.com:vogelpi/coco-alma.git alma -b fix-yosys-synth-template
```

Comment:
Can you give the exact SHA here? (Even just "The tip of this branch should be 123abcd")

Reply:
Done